src/NSubstitute/Substitute.cs

```diff
@@ -89,4 +90,19 @@ public static T ForPartsOf<T>(params object[] constructorArguments)
         var substituteFactory = SubstitutionContext.Current.SubstituteFactory;
         return (T)substituteFactory.CreatePartial([typeof(T)], constructorArguments);
     }
+
+        public static T ForTypeForwardingTo<T,TClass>(params object[] constructorArguments)
+            where T : class
+        {
+            var substituteFactory = SubstitutionContext.Current.SubstituteFactory;
+            return (T)substituteFactory.CreatePartial(new[] { typeof(T), typeof(TClass) }, constructorArguments);
+        }
+
```

Comment:
**question:** not sure if it is just the diff view but the code indenting looks non-standard here?